server/src/workspace/workspace-migration-runner/services/workspace-migration-enum.service.ts

```diff
@@ -0,0 +1,183 @@
+import { Injectable } from '@nestjs/common';
+
+import { QueryRunner } from 'typeorm';
+
+import { WorkspaceMigrationColumnAlter } from 'src/metadata/workspace-migration/workspace-migration.entity';
+
+@Injectable()
+export class WorkspaceMigrationEnumService {
```

Comment:
Should we prefix custom enums with "_"? @charlesBochet

Reply:
You mean because of table name? What if we add a custom enum on a standard object? As for the column name then yes we just to make sure we put an _ there when creating the migration